runtime/list_test.go

```diff
@@ -408,6 +408,8 @@ func TestListSort(t *testing.T) {
 		{args: wrapArgs(newTestList(true, false)), want: newTestList(false, true).ToObject()},
 		{args: wrapArgs(newTestList(1, 2, 0, 3)), want: newTestRange(4).ToObject()},
 		{args: wrapArgs(newTestRange(100)), want: newTestRange(100).ToObject()},
+		{args: wrapArgs(1), wantExc: mustCreateException(TypeErrorType, "'TestListSort' requires a 'list' object but received a \"int\"")},
+		{args: wrapArgs(newTestList("foo", "bar"), 2), wantExc: mustCreateException(TypeErrorType, "'TestListSort' requires 1 arguments")},
```

Comment:
The first test that was added to exercise 338-340 is fine.  It is this test I am wondering about:

```
{args: wrapArgs(1), wantExc: mustCreateException(TypeErrorType, "'TestListSort' requires a 'list' object but received a \"int\"")},
```

Doesn't that just test that the `fun` testing function throws an exception? i.e. it doesn't test any implementation code, just test code.